modules/diskq/qdisk.c

```diff
@@ -965,6 +963,12 @@ qdisk_reset_file_if_possible(QDisk *self)
     }
 }
 
+DiskQueueOptions *
+qdisk_get_options_reference(QDisk *self)
```

Comment:
From the name I would expect DiskQueueOptions is refcounted and this function increases the reference before returning. I suggest just calling qdisk_get_options.

Reply:
I had problem naming this function. :smile:
A "getter" would mean that you receive the value of the attribute and not the address of the attribute.
I am ok with `qdisk_get_options` though.

Reply:
+1 for `qdisk_get_options`.

Reply:
Okay, after our discussion I agree.